client/asset/dcr/dcr.go

```diff
@@ -164,6 +170,14 @@ var (
 				"limit orders without immediate time-in-force.",
 			IsBoolean: true,
 		},
+		{
+			Key:         "allowexternalapirequest",
+			DisplayName: "External fee rate estimates",
+			Description: "Allow fee rate estimation from a block explorer API. " +
+				"This is useful as a fallback for SPV wallets and RPC wallet th" +
+				"at have just recently been started.",
```

Comment:
Although this wrapping is valid and buck suggested it, it's a pain to grep for and confusing to read.  Longish code lines is just fine when they help readability. Let's just keep words together.

Also "RPC wallet" should be "RPC wallets", and <s>just</s>.